### ISSUE TITLE
Join forces with escompress?

### ISSUE THREAD

**Issue by ahutchings** [boopathi/babel-minify#1]
[escompress](https://github.com/escompress/escompress) uses a similar strategy of building a minifier on top of babel transforms. It may be worth consolidating your efforts.

I posted a similar issue in escompress's issue tracker here - https://github.com/escompress/escompress/issues/10

**Comment by boopathi** [comment]
Ah! Nice. This project started as more of a readme of list of plugins that you can use to optimize ES2015+ code. And the cli, and NodeAPI is merely a translation of options to the plugins list. But then I started writing a few plugins that I couldn't find.

Also, @hzoo mentioned on slack that he was working on some minifier as well which is not yet open sourced. So I think it's better to have multiple options for now, maybe? Anyway, because the plugins can be used in isolation with any host project/minifer, we can develop it wherever we want. Happy to see escompress 😄 and you guys had a go at mangler as well - Nice. I'll watch out for other plugins that you develop and avoid redundancies.

And all the plugins that I'm using and the options under which I'm using are listed here - 
https://github.com/boopathi/babel-minify/tree/master/packages/babel-minify#options

**Comment by ahutchings** [comment]
Just to be clear, I'm not affiliated with escompress, I just happened to run across it and babel-minify yesterday while looking for ES2015 minifiers. I understand your point about having multiple host projects for now - that makes sense, so that you guys can explore different public APIs/packaging strategies, etc. Maybe it's worth working together on just the mangler plugin?

Thanks for mentioning @hzoo's plans - looking forward to more progress in the ES2015 minifier space. I poked around in his repository list and found a couple more plugins you might mention in babel-minify:

https://github.com/hzoo/babel-plugin-minify-boolean-cast
https://github.com/hzoo/babel-plugin-minify-number-cast
